mender-convert-functions.sh

```diff
@@ -3,7 +3,10 @@
 # Partition alignment value in bytes (8MB).
 partition_alignment=8388608
 # Boot partition storage offset in bytes (8MB).
-vfat_storage_offset=8388608
+vfat_storage_offset_regular=8388608
+#erase_block=12582912
+# Boot partition storage offset in bytes (8MB).
+vfat_storage_offset_extended=12582912
```

Comment:
Why at 12 MB? If the partition alignment is defined as 8MB, it should be a multiple of that. Also update the comment, it still says 8MB

Reply:
Fixed